public/templates/payment-activation-content.html

```
@@ -0,0 +1,135 @@
+<!-- Payment Activation Content -->
+<table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation"
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)